bin/check-vpc-vpn.rb

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+require 'rubygems' if RUBY_VERSION < '1.9.0'
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+#
+# ===
+#
+# DESCRIPTION:
+#   This plugin checks VPC VPN connections to ensure they are up
+#
+# PLATFORMS:
+#   all
+#
+# DEPENDENCIES:
+#   sensu-plugin >= 1.5 Ruby gem
+#   aws-sdk Ruby gem
+#
```

Comment:
can you update the header to match the standard one at http://sensu-plugins.io/docs/developer_guidelines.html#coding-style and include usage.